src/main/webapp/app/admin/report-management/report-management.controller.js

```diff
@@ -31,36 +32,40 @@
         vm.goToBuildPage=goToBuildPage;
         vm.deleteReport=deleteReport;
         vm.updateReport=updateReport;
+        vm.serchReports=serchReports;
+        vm.searchUser=searchUser;
 
         activate();
         ///////////////////////////////////////
 
         function activate() {
-            getTotalScheduledReports(); 
-            getScheduledReports(); 
-            var cronstrue = window.cronstrue;   
+            getAccount(); 
+            getScheduledReports(vm.account.login,"","",""); 
+            var cronstrue = window.cronstrue; 
+             
         }
-
-        function getTotalScheduledReports(){
-            schedulerService.getScheduledReportsCount().then(
-              function(response) { 
-                vm.totalItems = response.data;
-                vm.queryCount = vm.totalItems;
-                vm.page = pagingParams.page;     
-              },
-              function(error) {
-                var info = {
-                    text: error.statusText,
-                    title: "Error"
-                }
-                $rootScope.showErrorSingleToast(info);
-              });
+        
+        function getAccount() {
+            vm.account = AccountDispatch.getAccount();
+            vm.isAdmin =  AccountDispatch.isAdmin();
         }
 
-        function getScheduledReports(){
-            schedulerService.getSchedulerReports(vm.itemsPerPage,pagingParams.page - 1).then(
+        function getScheduledReports(userName,reportName,startDate,endDate){
+            schedulerService.filterScheduledReports(userName,reportName,startDate,endDate,vm.itemsPerPage,pagingParams.page - 1).then(
               function(response) {
-                vm.reports=response.data;
+                if(response.data.records){
+                    vm.reports=response.data.records;
+                    vm.totalItems = response.data.totalRecords;
+                    vm.queryCount = vm.totalItems;
+                    vm.page = pagingParams.page;    
+                }
+                else{
+                    var info = {
+                    text: "No Records Found",
```

Comment:
error should come from notification center..it will be received by flair bi..send to UI